plugins/autoid/plugin.js

```diff
@@ -62,7 +62,7 @@
         editor.fire(EVENT_NAMES.ALL_IDS_COMPLETE);
       }
 
-      function findAllHeadings() {
+      function findAllHeadings(editor) {
         return editor.document.find('h1, h2, h3, h4, h5, h6');
```

Comment:
try editor.editable() instead of editor.document?  document actually means the whole HTML document (so either the iframe, ok for iframe mode, or the entire page that the inline one is on (not so good) )